src/scenes/AcceleratorRouter/DeliverableView.tsx

```diff
@@ -40,15 +41,15 @@ const DeliverableView = (props: Props): JSX.Element => {
 
   return (
     <Page title={<TitleBar {...viewProps} />} rightComponent={props.callToAction} crumbs={crumbs}>
-      <>
-        {props.isBusy && <BusySpinner />}
+      {props.isBusy && <BusySpinner />}
+      <Box display='flex' flexDirection='column' sx={{ flexGrow: 1 }}>
```

Comment:
- It looked like `BusySpinner` was defined outside of grid components/containers in other Views
- `flexGrow` was necessary here otherwise the page content wasn't stretching full-width (perhaps there's a preferred way to achieve this instead?)

Reply:
flexGrow is good 👍 
BusySpinner is absolutely positioned, good to be consistent anyways, thanks for that change.

Reply:
you can do `flexGrow={1}` instead of `sx` if you want